src/tools/management/commands/update_metrics.py

```diff
@@ -0,0 +1,49 @@
+from datetime import datetime
```

Comment:
We should deal with output_user_metrics.py and the bucket it writes to. Should we delete both? Or consolidate with this command as an option?

Reply:
Removed command